cardano_node_tests/tests/tests_conway/test_info.py

```diff
@@ -227,3 +230,23 @@ def test_info(
         governance_utils.check_vote_view(cluster_obj=cluster, vote_data=votes_drep[0])
         governance_utils.check_vote_view(cluster_obj=cluster, vote_data=votes_spo[0])
         reqc.cli022.success()
+
+        # Check dbsync
+        expected_votes_by_role = {
+            "ConstitutionalCommittee": [v.vote.name for v in votes_cc],
+            "DRep": [v.vote.name for v in votes_drep],
+            "SPO": [v.vote.name for v in votes_spo],
+        }
+
+        dbsync_votes = list(dbsync_queries.query_voting_procedure(txhash=vote_txid))
+
+        dbsync_votes_by_role: dict = {
+            "ConstitutionalCommittee": [],
+            "DRep": [],
+            "SPO": [],
+        }
+
+        for d_vote in dbsync_votes:
+            dbsync_votes_by_role[d_vote.voter_role].append(d_vote.vote.upper())
+
+        assert expected_votes_by_role == dbsync_votes_by_role, "Votes didn't match on dbsync"
```

Comment:
There should be a function for this in `dbsync_utils.py`. It can accept `VotedVotes` (move `VotedVotes` from `conway_common` to `governance_utils`).